javascripts/AppConfig.js

```diff
@@ -1,26 +1,26 @@
 'use strict';
 
-app.run(function(FIREBASE_CONFIG){
+app.run(function($location, $rootScope, FIREBASE_CONFIG, AuthService){
   firebase.initializeApp(FIREBASE_CONFIG);
 });
 
 app.config(function($routeProvider){
   $routeProvider
     .when("/contacts/favorites", {
       templateUrl: 'partials/favorites.html',
-      controller: 'FavoritesCtrl'
```

Comment:
you don't need to be adding the commas here, since the last key value pair of an object shouldn't have a comma